README

```diff
@@ -49,11 +40,29 @@ plugins to build large scale web applications.
       |-- http/ .......... The actual HTTP protocol implementation
       |---http2/ ......... HTTP/2 implementation
       |-- logging/ ....... Flexible logging
-  |-- README ............. intro, links, build info
-  |-- README-EC2 ......... info on EC2 support
-  |-- REVIEWERS .......... (incomplete) list of areas with committer interest
-  |-- LICENSE ............ full license text
-  `-- NOTICE ............. copyright notices
+  |-- rc/ ................ Installation programs and scripts
+  |-- src/ ............... Source for all the main binaries / applications
+      |-- traffic_cache_tool/
+      |-- traffic_crashlog/
+      |-- traffic_ctl/ ... traffic_ctl command line management application
+      |-- traffic_layout/
+      |-- traffic_logcat/
+      |-- traffic_logstats/
+      |-- traffic_manager/
+      |-- traffic_server/
+      |-- traffic_top/ ... traffic_top application
```

Comment:
This "help text" seems redundant.  :-). Instead, maybe something like:

traffic_ctl/ ... command line management tool
...
traffic_top ... top like tool for viewing current traffic metrics

Reply:
absolutely

Reply:
Do you think it's better to take these useless texts out from these cmd tools or just leave the two text there and make the "help text" useful? Another option is to add "help text" to all of them. @zwoop